test/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs

```diff
@@ -11,6 +11,7 @@
 using System.Text.Json.Serialization;
 using System.Threading.Tasks;
 using Microsoft.DotNet.MsiInstallerTests.Framework;
+using Microsoft.DotNet.Workloads.Workload;
```

Comment:
we should probably sort these usings

Reply:
I've removed the unused usings.  I think the System namespace should generally be sorted before other namespaces in the usings, so I think other than that they are sorted.